sdk/python/kfp_tekton/compiler/compiler.py

```diff
@@ -38,6 +38,36 @@
 from kfp_tekton import tekton_api_version
 
 
+def _get_super_condition_template():
+
+  python_string = "python -c 'import sys;input1=str.rstrip(sys.argv[1]);input2=str.rstrip(sys.argv[2]);" \
```

Comment:
Another way you can make it cleaner is to use the condition script instead of args and commands.
https://github.com/tektoncd/pipeline/blob/master/examples/v1beta1/pipelineruns/conditional-pipelinerun-with-same-condition-refer.yaml#L13-L21

Then you can use the textwrap library to put things into a single string.

Reply:
https://github.com/kubeflow/kfp-tekton/blob/master/sdk/python/kfp_tekton/compiler/_op_to_template.py#L308-L311